src/Blocks/components/form/assets/utils.js

```diff
@@ -272,15 +272,15 @@ export class Utils {
 			return;
 		}
 
-		messageContainer?.classList?.add(this.state.getStateSelector('isActive'));
-		messageContainer.dataset.status = status;
-
 		// Scroll to msg if the condition is matched.
 		if (status === 'success') {
 			if (this.state.getStateFormGlobalMsgHideOnSuccess(formId)) {
 				return;
 			}
 
+			messageContainer?.classList?.add(this.state.getStateSelector('isActive'));
+			messageContainer.dataset.status = status;
```

Comment:
```suggestion
			messageContainer?.dataset.status = status;
```

Reply:
you can't put ?. if you are setting values.

Reply:
Lets just say it was a long week and forget about this blunder 🙈

Reply:
![images](https://github.com/infinum/eightshift-forms/assets/23283324/b21ba0a2-4d53-490e-8856-768c2d9952ea)